boards/sim/sim/sim/configs/adb/defconfig

```diff
@@ -26,6 +26,9 @@ CONFIG_INIT_ENTRYPOINT="nsh_main"
 CONFIG_LIBC_DLFCN=y
 CONFIG_LIBUV=y
 CONFIG_LIBUV_UTILS_TEST=y
+CONFIG_MM_BACKTRACE=8
+CONFIG_MM_HEAP_MEMPOOL_THRESHOLD=256
+CONFIG_MM_TLSF_MANAGER=y
```

Comment:
@xiaoxiang781216 how does TLSF improve CI coverage?

Reply:
TLFS isn't enabled in any mainline defconfig, so this patch could catch TLFS related error now. For example, these two patch is found by CI:
https://github.com/apache/nuttx/pull/9412/commits/0dd2039dfd860fe1ca7704bc7186531011d2f9be
https://github.com/apache/nuttx/pull/9412/commits/7fb07452cdd5667d9f94b22af5d69e8d63f3c9d1